setup.py

```diff
@@ -36,7 +35,6 @@
     "Intended Audience :: Science/Research",
     "Development Status :: 5 - Production/Stable",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3.5",
```

Comment:
@darcymason in the setup.py, should also include a "python requires" field:

https://stackoverflow.com/a/48777286

That way, if someone has an unsupported python version on their machine in will instead install that last version that was available that was supported. That way, pip won't install this version, it'll install the older version if the user has, for example, Python 2.7.

Reply:
Thanks, Simon, I hadn't heard of that option.

Reply:
> Thanks, Simon, I hadn't heard of that option.

My pleasure :)